### DISCUSSION TITLE
Weather Alerts

### DISCUSSION THREAD

**Discussion by 0-5-0** [open-meteo/open-meteo#183] [Ideas]
Any chance of adding national weather service alerts? NWS agencies will often publish storm warnings and this would be great to have accessible through the Open-Meteo API.

I couldn't see it mentionned in other discussions, issues or as a feature on the Open-Meteo website.

Thank you for making this API accessible.

**Comment by phillipseamore** [comment]
For Europe I would suggest using the official Metoalarm pub/sub https://feeds.meteoalarm.org/

When you've setup a subscriptions correctly you will be notified just about immediately once an alert is issued, updated or canceled.

**Comment by patrick-zippenfenig** [comment]
Thanks @phillipseamore for the meteoalarm suggestion! There is also https://www.alert-hub.org which lists many available CAP feeds.

For the next months, I do not plan to include alerts. Maybe this could be done at a later stage

**Comment by holle75** [comment]
Just want to second Alerts. The only missing thing in open-meteo.

**Comment by ZapsharkRS** [comment]
So I help a non-profit out on their tech needs. I use a mix between IEMBot, NWS Weather API, and the XML's for North America. With IEMBot, you can tap into an XMPP feed. With the Weather API and the XML's they provide, I have that set on a corn to pull once a minute. IEMBot is really the only reliable means for instantaneous alerts. NWS Weather API is VERY prone to issues like delays and random outages. XML has been pretty reliable and pretty quick to pull. It's just a lot more data.

**Reply by patrick-zippenfenig** [reply]
Is this only available for alerts in the US or for multiple countries? The main issue is to "unify" so many data sources reliably.

**Comment by ZapsharkRS** [comment]
> Is this only available for alerts in the US or for multiple countries? The main issue is to "unify" so many data sources reliably.

Just for the US. The Non-Profit I work with only does localized weather for a state in the US, but we also provide a means of national alerts as a courtesy to the people as they travel across the US. 

Unifying these alerts reliably wouldn't be too hard based on geojson features these alerts give out, similar to what I've heard comes from other countries across Europe at least. Link to NWS for the US Alerts that I use: https://www.weather.gov/documentation/services-web-api

**Comment by 0-5-0** [comment]
Breezy Weather is using WMO as a source for weather alerts (see https://github.com/breezy-weather/breezy-weather/discussions/1557), would WMO be a good source to use for Open-Meteo weather alerts https://severeweather.wmo.int/? 

You would hopefully have global coverage without needing to re-invent the wheel since WMO appears to be doing a good job probing all the various providers already.

I'm wondering if they might need to be contacted first to make sure that the data could be used in open-meteo. It states here https://wmo.int/copyright "You may freely download and copy the WMO material contained on this website for your personal, non-commercial use, without any right to resell, redistribute, compile or create derivative works therefrom, subject to more specific restrictions that may apply to specific materials." ... however I couldn't imagine they would refuse any type of agreement with Open-Meteo, because Open-Meteo is awesome ;).

I guess for commercial apps that use Open-Meteo it becomes less ideal, I'm not sure if they would mind though since Open-Meteo itself still makes their API/information freely available to all/anyone. WMO's mandate states: "WMO provides the framework for international cooperation to advance meteorological, climatological, hydrological, and related environmental services, to improve well-being of all" (https://wmo.int/about-wmo/our-mandate).

https://severeweather.wmo.int/about.html

Otherwise if WMO doesn't work then I guess it might be a good source of individual feeds: https://severeweather.wmo.int/sources.html

**Comment by asimonigh** [comment]
Same need here ✋
